Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/polygon_soup_to_polygon_mesh.h

```diff
@@ -67,13 +68,29 @@ class Polygon_soup_to_polygon_mesh
       _polygons(polygons)
   { }
 
-  void operator()(PM& pmesh)
+  void operator()(PM& pmesh, const bool insert_isolated_vertices = true)
```

Comment:
I am not a big fan of Boolean parameters. When one see a call `polygon_soup_to_polygon_mesh(mesh, true)`, its hard to know, and to remember, if the `true` means "keep the isolated vertices" or "remove the isolated vertices". Explicit enumerators allows to see call like `polygon_soup_to_polygon_mesh(mesh, REMOVE_ISOLATED_VERTICES)` which is a lot more explicit.

That is just a remark, for a later time.